### ISSUE TITLE
[MainUI] Blank page when access is from myopenhab after getting admin rights

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-webui#907]
## The problem

Opening MainUI in admin mode from myopehab is not working, only a blank page is displayed.

## Expected behavior

I should have access to MainUI in admin mode from myopenhab.

## Steps to reproduce

1. From a Web browser (Firefox in my case), log in myopenhab with your credentials
2. Open the Main UI => MainUI is ok but you have not admin rights.
3. Try to unlock the admin rights by providing login/password (I used the login/password I initally created when installing the OH server) for https://home.myopenhab.org => a blank page is then displayed.
4. Connecting to the server from the local network, I can see that a session has been created for https://home.myopenhab.org
5. If I remove this session, it is again possible to have MainUI working from myopenhab but of course with no admin rights.

My server is OH3 build 2192.

@ghys : is it expected ? Is there something I don't understand ? Is there a special server setting to change to aloow that ?

**Comment by lolodomo** [comment]
@ghys : in fact, the problem is very probably more a REST API problem but I am not sure.

**Comment by candiesdoodle** [comment]
Related . Check your server logs.

https://github.com/openhab/openhab-webui/issues/888

**Comment by lolodomo** [comment]
Oh, you're right.
```
2021-02-14 18:38:37.892 [WARN ] [io.openhabcloud.internal.CloudClient] - Unsupported request method HEAD
```
I closed my issue as it is a duplicate of #888
